src/definitions/modules/dropdown.less

```diff
@@ -1574,8 +1580,7 @@ select.ui.dropdown {
         Floating
     --------------- */
 
-    .ui.floating.dropdown > .menu {
-        left: 0;
+    .ui.floating.dropdown .menu {
```

Comment:
The direct child selector > was introduced by #2284
This change re-introduces that behavior again (see your jsfiddle)
![image](https://github.com/fomantic/Fomantic-UI/assets/18379884/db1a4f14-b47d-4892-8dc1-e9e6132b0ea0)

Reply:
I don't see that in my fiddle and local version.

Reply:
Sorry. You're right. I was checking the dropdown without scrollbar.